### ISSUE TITLE
start up error

### ISSUE THREAD

**Issue by jiahaoxiang2000** [microsoft/vscode-mermAId#80]
## Issue Description
When starting up the vscode-mermAId extension, an error message is displayed indicating that the extension cannot be activated due to an unloaded dependency on the GitHub Copilot Chat extension.

### Error Message
Cannot activate the 'vscode-mermAId' extension because it depends on the 'GitHub Copilot Chat' extension, which is not loaded. Would you like to reload the window to load the extension?
## Steps to Reproduce
Install the vscode-mermAId extension.
Ensure the GitHub Copilot Chat extension is also installed.
Start or reload the VS Code window.
## Expected Behavior
The vscode-mermAId extension should activate without any errors if all dependencies, including the GitHub Copilot Chat extension, are correctly loaded.

## Actual Behavior
An error message is displayed stating that the vscode-mermAId extension cannot be activated because the GitHub Copilot Chat extension is not loaded, despite being installed.

**Comment by joshspicer** [comment]
Thank you for taking the time to create an issue.  Could you please confirm you have the latest GitHub Copilot Chat extension installed?  We're using some cutting edge features in the chat extension which depend on the newest builds 🙂   

If you're still able to repro, do you mind sharing the versions of both the mermAId and chat extensions so I can dig a bit deeper?

**Comment by jiahaoxiang2000** [comment]
> If you're still able to repro, do you mind sharing the versions of both the mermAId and chat extensions so I can dig a bit deeper?

@joshspicer, yes, the GitHub Copilot Chat extension is v0.22.0 (2024-10-29), so it is the latest version. The mermAId extension is v0.0.2024103001 (pre-release). Thank you for your time and help.
